### DISCUSSION TITLE
PyPal機器人猜猜樂

### DISCUSSION THREAD

**Discussion by skyling0299** [1101-GS3073-C/Python-Community-GS3073-C#563] [Ideas]
發現老師update了這個機器人的code
目前是只要你發文或討論，機器人都會自動幫你看你的文字有沒有原創性

但這個原創性有很多定義，想問大家目前試出哪幾種原創性了呢~~
或是機器人是如何判斷你的發言有原創性

歡迎討論和分享你的發現

**Comment by HsuChen88** [comment]
我感覺我的發言都很有原創性，但他一直傳訊息提醒我，讓我都要懷疑自己了。

**Comment by HsuChen88** [comment]
目前得到最好的回覆是「棒棒，你是大文豪。」不知為何被誇獎後也開心不起來。

**Reply by chia-kai-chang** [reply]
QQ

**Comment by awehao** [comment]
什麼意思
更
完拉，好多紅色的
再更
要綠色的才有3分歐

**Reply by skyling0299** [reply]
綠色只有兩分喔哈哈~今天又更新了
我怎麼覺得老師好像...期末變閒了就開始開發這個了

**Reply by awehao** [reply]
歐我說得是發文的部分，謝謝你的分享!我原本都不知道這個功能诶，其實還挺有趣的

**Reply by chia-kai-chang** [reply]
感謝同學們的捧場 哈哈~ 主要是希望大家可以多在Github討論區多跟我互動，不然寒假老師一個人在辦公室會覺得很無聊的~

**Comment by skyling0299** [comment]
一覺睡起來發現老師又更新了機器人檢測的方式，目前為止字數似乎還是很重要的一個part。如果不知道寫甚麼就多寫一點吧
另外如果是補充同學介紹的專有名詞也很容易拿高分~

**Comment by Raywei0214** [comment]
好像隨便瞎掰就有黃色了，像我現在打的這句，你可能看不出什麼所以然，我也覺得非常無聊，但我才發現要70點才會滿分，所以就這樣瞜

**Reply by chia-kai-chang** [reply]
這不就來了嗎~ 👀

**Comment by Raywei0214** [comment]
但我覺得最大的困擾就是他除了寄telagram之外還會寄到gmai，看了還挺煩躁的

**Reply by chia-kai-chang** [reply]
我程式沒有設計到mail到同學的信箱，可以轉寄給我一下嗎? 謝啦
寄到： wearetheborg@ncuedu.tw

謝謝喔

**Reply by chia-kai-chang** [reply]
啊我知道了，email是從Github寄出去的，不是從PyPal寄出去的。

同學可以把它關掉，就不會寄通知信了。

![image](https://user-images.githubusercontent.com/1768891/211012640-17881bbf-5204-41a1-a48d-d74716614d5d.png)

**Comment by magicmicheal** [comment]
不知道大家是靠著一堆黃色拿分，還是靠著一半數量的綠色拿分呢？我目前的觀察是字多打一點綠色的機率就高一點所以我要開始講廢話衝字數了

**Reply by chia-kai-chang** [reply]
剛好也可以培養大家寫文章衝字數的能力

不過要言之有物喔~ 我們一起加油 

老師最近也在趕國科會計畫~ 我們一起寫文章~

**Comment by awehao** [comment]
看到有人說"好像隨便瞎掰就有黃色了，像我現在打的這句，你可能看不出什麼所以然，我也覺得非常無聊，但我才發現要70點才會滿分，所以就這樣瞜"，我也在想，那我這樣又是如何呢，若是我在這一句的基礎下加入更多個人的想法，那結果是不是會不同呢，這點值得我們深思和探討，這就是學習的樂趣所在，同時也顯現出了AI的不足之處

**Comment by bww0401** [comment]
我目前打了十篇回復的感想是，如果打得內容超過一行，機器人基本上都會判定是綠色，但這只是我的推測，也有可能機器人是抓取特定字眼也說不定，我要繼續去測試它了: )

**Reply by chia-kai-chang** [reply]
讚喔~ 繼續測試~ 發揮實驗精神~
